<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
    ~~~~~~~~~~~~~~~~^
        recipe,
        ^^^^^^^
        fix=fix,
        ^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
    ~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/check_syntax.py", line 36, in check_recipe
    has_constraints = constraints.search(spec)
TypeError: expected string or bytes-like object, got 'NoneType'

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_d5o5j56xw9/clone` failed. (See above for error)
</details>

--- a/recipe/bld.bat
+++ b/recipe/bld.bat
@@ -1,22 +1,49 @@
-@echo on
+setlocal enabledelayedexpansion
 
 set "STDLIB_DIR=%PREFIX%\Lib;%PREFIX%;%LIBRARY_BIN%"
-%PYTHON% -m pip install --no-deps --no-build-isolation . -vv
-if %ERRORLEVEL% neq 0 exit 1
-echo "sleeping for 15"
-%PYTHON% -c "import time; time.sleep(15)"
-if %ERRORLEVEL% neq 0 exit 1
-echo "Copying over stray DLLS"
-if %ERRORLEVEL% neq 0 exit 1
+%PYTHON% setup.py install --record=record.txt
 
-dir %PREFIX%\Lib\site-packages\win32\py*.dll
+copy %PREFIX%\Lib\site-packages\pythonwin\*.pyd %PREFIX%\Lib\site-packages\win32
+if errorlevel 1 exit /b 1
+copy %PREFIX%\Lib\site-packages\pythonwin\*.dll %PREFIX%\Lib\site-packages\win32
+if errorlevel 1 exit /b 1
+copy %PREFIX%\Lib\site-packages\pythonwin\*.dll %LIBRARY_BIN%\
+if errorlevel 1 exit /b 1
+
+mkdir %PREFIX%\Library\bin
+copy %PREFIX%\Lib\site-packages\win32\*.dll %PREFIX%\Library\bin\
+if errorlevel 1 exit /b 1
+
+:: Fix for https://sourceforge.net/p/pywin32/mailman/message/29498528/
+:: although on that bug report Glenn Linderman claims this fix does
+:: not work, it seems to work fine. I attempted a fix in the source
+:: code (and the upstream developers have clearly thought about it)
+:: but the fact is win32api auto-imports pywintypes??.dll as can be
+:: seen from:
+:: ntldd.exe /c/aroot/stage/Lib/site-packages/win32/win32api.pyd
+::         pywintypes36.dll => not found
+:: due to: win32/src/PyWinTypes.h:
+:: define PYWINTYPES_EXPORT __declspec(dllimport)
+:: .. and ..
+:: pragma comment(lib,"pywintypes.lib")
+::  .. and then (at least): win32/src/win32apimodule.cpp
+:: PYWINTYPES_EXPORT PyObject *PyWin_NewUnicode(PyObject *self, PyObject *args);
+:: therefore copying is the only recourse. At first glance, it may
+:: seem that moving these DLLs would work, but _win32sysloader.cpp
+:: expects to find two DLLs in site-packages/pywin32_system32
+:: My attempted fix is in import-pywintypes-from-win32api.patch
+:: An actual fix would be to make win32api a Python module that
+:: first imports pywintypes and after that imports _win32api
 copy %PREFIX%\Lib\site-packages\pywin32_system32\*.dll %PREFIX%\Lib\site-packages\win32\
-dir %PREFIX%\Lib\site-packages\win32\py*.dll
-if %ERRORLEVEL% neq 0 exit 1
+if errorlevel 1 exit /b 1
 
-dir %LIBRARY_BIN%\py*.dll
+:: The post install script had code that remove those files. See patch 0004.
 copy %PREFIX%\Lib\site-packages\pywin32_system32\*.dll %LIBRARY_BIN%\
-if %ERRORLEVEL% neq 0 exit 1
+if errorlevel 1 exit /b 1
 
-dir %LIBRARY_BIN%\py*.dll
-if %ERRORLEVEL% neq 0 exit 1
+echo "Contents of Library\bin:"
+dir /b %PREFIX%\Library\bin\*.dll
+echo "Contents of site-packages\win32:"
+dir /b %PREFIX%\Lib\site-packages\win32\*.dll
+
+exit 0

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -66,7 +66,6 @@ extra:
   recipe-maintainers:
     - mingwandroid
     - nicoddemus
-    - carlodri
-  # Explicitly skip license linting because of the license complexity
+    - carlodri  # Explicitly skip license linting because of the license complexity
   skip-lints:
     - missing_license_family

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -49,9 +49,13 @@ test:
 
 about:
   home: https://github.com/mhammond/pywin32
-  license: PSF-2.0
-  license_file: Pythonwin/License.txt
-  license_family: PSF
+  license: HPND AND BSD-3-Clause AND LGPL-2.1-or-later AND MIT
+  license_file:
+    - Pythonwin/License.txt  # BSD-3-CLause
+    - Pythonwin/Scintilla/License.txt  # MIT
+    - win32/License.txt  # BSD-3-CLause
+    - com/License.txt  # BSD-3-CLause
+    - adodbapi/license.txt  # LGPL-2.1-or-later  summary: Python extensions for Windows
   summary: Python extensions for Windows
   description: |
     A set of extension modules that provides access to many of the Windows API functions.
@@ -63,3 +67,5 @@ extra:
     - mingwandroid
     - nicoddemus
     - carlodri
+skip-lints:
+    - missing_license_family

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -67,5 +67,6 @@ extra:
     - mingwandroid
     - nicoddemus
     - carlodri
-skip-lints:
+  # Explicitly skip license linting because of the license complexity
+  skip-lints:
     - missing_license_family

--- a/recipe/run_test.py
+++ b/recipe/run_test.py
@@ -64,7 +64,6 @@ else:
             )
         )
     )
-
     library_bin = os.environ["LIBRARY_BIN"]
     pythoncom_filename = os.path.join(library_bin, f"pythoncom{conda_py}.dll")
     pywintypes_filename = os.path.join(library_bin, f"pywintypes{conda_py}.dll")

